mimic/test/test_auth.py

```diff
@@ -915,6 +916,114 @@ def test_response_for_validate_impersonated_token_multiple_users(self):
         self.assertEqual(json_body6["access"]["RAX-AUTH:impersonator"]["name"],
                          json_body1["access"]["user"]["name"])
 
+    def test_response_for_validate_token_with_maas_admin_role(self):
+        """
+        Test to verify :func: `validate_token` when the token_id provided
+        is of an maas admin user specified in `mimic_presets`.
+        """
+        core = MimicCore(Clock(), [ExampleAPI()])
+        root = MimicRoot(core).app.resource()
```

Comment:
Given that each of these tests starts with *exactly* these two lines, maybe you could factor out just a simple function so we could do `core, root = core_and_example_root()`?